ironfish/src/wallet/account/account.ts

```diff
@@ -76,6 +82,15 @@ export class Account {
     this.walletDb = walletDb
     this.version = version ?? 1
     this.createdAt = createdAt
+
+    if (multiSigKeys !== undefined) {
+      Assert.isNull(
+        this.spendingKey,
+        'An account cannot have its own spending key if it is part of a signing group.',
```

Comment:
nit: shorten the error message "Multi sig wallet cannot have a spending key"?

Reply:
I'm actually thinking that it may be better not to assert in the constructor and to handle this logic at import/account creation 🤔